rust/pylance/src/scanner.rs

```diff
@@ -0,0 +1,93 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+use pyo3::prelude::*;
+use pyo3::{pyclass, PyObject, PyResult};
+/// This a Scanner implementation which will being associated
+/// with a Pyarrow Scanner so duckdb can be tricked to call
+/// it's methods during scans
+use std::sync::Arc;
```

Comment:
Lets order uses in section? `std` / `thirdparty` / `lance`?

Reply:
is there a cargo fmt setting for this?